pkgs/by-name/ay/ayugram-desktop/package.nix

```diff
@@ -0,0 +1,50 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitHub,
+  telegram-desktop,
+}:
+let
+  mainProgram = if stdenv.isLinux then "ayugram-desktop" else "Ayugram";
+in
+(telegram-desktop.override { inherit mainProgram; }).overrideAttrs (
+  finalAttrs: previousAttrs: {
+    pname = "ayugram-desktop";
+    version = "4.16.8";
+
+    src = fetchFromGitHub {
+      owner = "AyuGram";
+      repo = "AyuGramDesktop";
+      rev = "v${finalAttrs.version}";
+      fetchSubmodules = true;
+      hash = "sha256-HrvqENRRyRzTDUUgzAHPBwNVo5dDTUsGIFOH75RQes0=";
+    };
+
+    # Since the original desktop file is for Flatpak, we need to fix it.
+    postInstall = lib.optionalString stdenv.isLinux ''
+      # Rudiment: related functionality is disabled by disabling the auto-updater
+      # and it breaks the .desktop file in Aylur's Gtk Shell
+      # (with it, it causes the application to not be seen by the app launcher).
+      # https://github.com/AyuGram/AyuGramDesktop/blob/5566a8ca0abe448a7f1865222b64b68ed735ee07/Telegram/SourceFiles/platform/linux/specific_linux.cpp#L455
+      substituteInPlace $out/share/applications/com.ayugram.desktop.desktop \
+        --replace-fail 'Exec=DESKTOPINTEGRATION=1 ' 'Exec='
+
+      # Since we aren't in Flatpak, "DBusActivatable" has no unit to
+      # activate and it causes the .desktop file to show the error "Could not activate remote peer
+      # 'com.ayugram.desktop': unit failed" (at least on KDE6).
+      substituteInPlace $out/share/applications/com.ayugram.desktop.desktop \
+        --replace-fail 'DBusActivatable=true' '# DBusActivatable=true'
+    '';
+
+    meta = {
+      description = "AyuGram Desktop messaging app";
+      longDescription = "A desktop Telegram client with great customization and Ghost mode.";
+      license = lib.licenses.gpl3Only;
+      platforms = lib.platforms.all;
+      homepage = "https://t.me/ayugram1338";
```

Comment:
```suggestion
      homepage = "https://github.com/AyuGram/AyuGramDesktop";
```

Reply:
I added the requested change, but could you please elaborate why you suggested like to GitHub repository? The “about” description in GH uses the original link. 
![image](https://github.com/NixOS/nixpkgs/assets/70033218/5ccee05d-e75d-47ac-8c86-ce176a855bac)
The other options are:
* [`ayugram.one`](https://ayugram.one) — currently redirects to [AyuGram organization on GitHub](https://github.com/AyuGram)
* [`ayugram1338.t.me`](https://ayugram1338.t.me) — just another way to write [`t.me/ayugram1338](https://t.me/ayugram1338)